Gems/MotionMatching/Code/Tests/MinMaxScalerTests.cpp

```diff
@@ -0,0 +1,125 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#include <Fixture.h>
+#include <FeatureMatrixMinMaxScaler.h>
+
+namespace EMotionFX::MotionMatching
+{
+    class MinMaxScalerFixture
+        : public Fixture
+    {
+    public:
+        void SetUp() override
+        {
+            Fixture::SetUp();
+
+            // Construct 3x3 matrix:
+            // 1 2 3
+            // 4 5 6
+            // 7 8 9
+            m_featureMatrix.resize(3, 3);
+
+            float counter = 1.0f;
+            for (size_t row = 0; row < 3; ++row)
+            {
+                for (size_t column = 0; column < 3; ++column)
+                {
+                    m_featureMatrix(row, column) = counter;
+                    counter++;
+                }
+            }
+        }
+
+        FeatureMatrix m_featureMatrix;
+    };
+
+    TEST_F(MinMaxScalerFixture, MinMaxValues)
+    {
+        FeatureMatrix m;
+        m.resize(3, 3);
+        m(0,0) = 0.0f;    m(0,1) =-1.0f;    m(0,2) = 9.0f;
```

Comment:
Strictly speaking I think the coding standards discourage multiple statements per line, although I think it makes sense in this case.